src/settings-ui/Settings.UI/Strings/en-us/Resources.resw

```diff
@@ -1354,10 +1353,10 @@ Made with 💗 by Microsoft and the PowerToys community.</value>
     <value>PowerToys Run is a quick launcher for power users that contains some additional features without sacrificing performance.</value>
   </data>
   <data name="Oobe_ShortcutGuide.Description" xml:space="preserve">
-    <value>Shortcut Guide presents the user with a listing of available shortcuts for the current state of the desktop.</value>
+    <value>Shortcut Guide presents you a listing of available shortcuts for the current state of the desktop.</value>
```

Comment:
Nice! This is inline with the Windows Writing Style :).. we should use 'you' everywhere instead of 'user'